conversions/octal_to_binary.py

```diff
@@ -0,0 +1,55 @@
+"""
+* Author: Bama Charan Chhandogi (https://github.com/BamaCharanChhandogi)
+* Description: Convert a Octal number to Binary.
+
+reference for better understand
+
+URL: https://en.wikipedia.org/wiki/Binary_number
+URL: https://en.wikipedia.org/wiki/Octal
```

Comment:
Formatting and grammar fixes
```suggestion
References for better understanding:
https://en.wikipedia.org/wiki/Binary_number
https://en.wikipedia.org/wiki/Octal
```